django/db/models/base.py

```diff
@@ -1335,6 +1335,12 @@ def _check_field_name_clashes(cls):
 
         # Check that fields defined in the model don't clash with fields from
         # parents.
+        # Also include Shadow fields of parent class
```

Comment:
`Also include shadowed fields of the parent class.` (can move to previous line I think)